fighthealthinsurance/settings.py

```diff
@@ -35,6 +36,7 @@
 
 from rest_framework.authentication import SessionAuthentication, BasicAuthentication
 
+TESTING = 'test' in sys.argv
```

Comment:
This seems like a weird way to determine testing, maybe just put this in the test envs as true and default as false?